apps/server/src/modules/learnroom/uc/dashboard.uc.spec.ts

```diff
@@ -113,7 +102,7 @@ describe('dashboard uc', () => {
 			const syncSpy = jest.spyOn(dashboard, 'setLearnRooms');
 			const persistSpy = jest.spyOn(repo, 'persistAndFlush');
 
-			const result = await service.getUsersDashboard('userId', true);
+			const result = await service.getUsersDashboard('userId');
 
 			expect(result instanceof DashboardEntity).toEqual(true);
 			expect(dashboardRepoSpy).toHaveBeenCalledWith('userId');
```

Comment:
The mocks for `findAllForSubstituteTeacher` in lines 134 and 162 can be removed.

Reply:
removed